<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagePreviewer</class>
 <widget class="QWidget" name="MessagePreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="m_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="MessageTextBrowser" name="m_txtMessage">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SearchTextWidget" name="m_searchWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>messagetextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>SearchTextWidget</class>
   <extends>QWidget</extends>
   <header>searchtextwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
